stash_engine/app/mailers/stash_engine/user_mailer.rb

```diff
@@ -38,6 +38,17 @@ def error_report(resource, error)
            subject: "#{rails_env} Submitting dataset \"#{@title}\" (doi:#{@identifier_value}) failed")
     end
 
+    def dependency_offline(dependency)
+      return unless dependency.present?
+      @dependency = dependency
+      @url = status_dashboard_url
+
+      @submission_error_emails = ['brian.riley@ucop.edu']
```

Comment:
This email address should be in a config file.

Reply:
ah, was just using that for testing. Removed it so that it will send out to you and scott instead